lib/src/admob_banner_size.dart

```diff
@@ -15,10 +15,16 @@ class AdmobBannerSize {
       AdmobBannerSize(width: 468, height: 60, name: 'FULL_BANNER');
   static const AdmobBannerSize LEADERBOARD =
       AdmobBannerSize(width: 728, height: 90, name: 'LEADERBOARD');
-  AdmobBannerSize.SMART_BANNER(BuildContext context):
-        width = MediaQuery.of(context).size.width.toInt(), height = -2, name = 'SMART_BANNER';
-  AdmobBannerSize.ADAPTIVE_BANNER({@required int width}):
-      width = width, height = -2, name = 'ADAPTIVE_BANNER';
+  static const AdmobBannerSize PIXEL =
+      AdmobBannerSize(width: 1, height: 1, name: 'PIXEL');
```

Comment:
Sorry, I'm not following why you are setting a banner size of 1x1?

Reply:
As Stated in the PR, we use 1x1 AdUnits to pass messages and data to build custom ads experience within our application.
So one  example we do is that i can pass data to change for example the color of my background when the ad is loaded.
We don't need to render anything through the view so we use the smallest possible unit :)